### ISSUE TITLE
[Bug][A11Y][Android] Switch : remove the useless focus on switch

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#364]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S22

### OS Version

.

### App version

0.60

### Describe the issue

With VoiceOver or external keyboard, the focus goes twice on the switch component 

**Group label+switch is correct:**
<img width="336" height="73" alt="" src="https://github.com/user-attachments/assets/1ff328ce-4c5d-48ed-b625-726989f7ecb5" />

**Switch-only focus is useless:**
<img width="289" height="57" alt="" src="https://github.com/user-attachments/assets/97538e6e-cd6f-4ab8-8c0b-8265fe277573" />

### Expected Behavior

Remove the switch-only focus

**Comment by pya35** [comment]
linked to #49

**Comment by nouha06** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by B3nz01d** [comment]
Closed during Sprint # 23

**Comment by nouha06** [comment]
New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 438
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 438
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)
